sympy/utilities/iterables.py

```diff
@@ -2034,18 +2060,21 @@ def generate_oriented_forest(n):
     also be described as a disjoint union of trees, which are graphs in which
     any two vertices are connected by exactly one simple path.
 
-    Reference:
-    [1] T. Beyer and S.M. Hedetniemi: constant time generation of \
-        rooted trees, SIAM J. Computing Vol. 9, No. 4, November 1980
-    [2] https://stackoverflow.com/questions/1633833/oriented-forest-taocp-algorithm-in-python
-
     Examples
     ========
 
     >>> from sympy.utilities.iterables import generate_oriented_forest
     >>> list(generate_oriented_forest(4))
     [[0, 1, 2, 3], [0, 1, 2, 2], [0, 1, 2, 1], [0, 1, 2, 0], \
     [0, 1, 1, 1], [0, 1, 1, 0], [0, 1, 0, 1], [0, 1, 0, 0], [0, 0, 0, 0]]
+
+    References
+    ==========
+    .. [1] T. Beyer and S.M. Hedetniemi: constant time generation of
```

Comment:
See the first part of https://github.com/sympy/sympy/pull/16143#discussion_r261837584.